src/index.scss

```diff
@@ -34,19 +45,47 @@ body {
   padding: 0.5rem;
 }
 
-.main {
-  display: flex;
-  justify-content: space-around;
+.main__h {
+  font-size: 2.5rem;
+  margin-bottom: 2rem;
 }
 
-.ul {
-  list-style: none;
-  border: 3px solid $primary-black;
+button {
+  background-color: $third-maroon;
+  color: $primary-white;
+  border: none;
+  border-radius: 0.5rem;
+  font-size: 1.5rem;
+  font-family: $primary-font;
+}
```

Comment:
- [ ] [OPTIONAL] For your buttons, the color combination for both background and text is not great for accessibility. Users will find it hard to make out the button texts. Please consider using your colors in a more accessible way 🙏

Reply:
Hi @codepantha,

Thanks for that feedback, I'll put that into practice 👍

Regards!!!